README.md

```diff
@@ -12,7 +12,7 @@ git2 = "0.10"
 ## Rust version requirements
 
 git2-rs works with stable Rust, and typically works with the most recent prior
-stable release as well. Check [azure-pipelines.yml](azure-pipelines.yml) to see the oldest
+stable release as well. Check [GitHub Actions](https://github.com/rust-lang/git2-rs/actions) to see the oldest
```

Comment:
Thanks!  I think the intent here is to point to the source file so that one can see the `MSRV` value, which can't be quickly discerned from the Actions page.  Maybe something like this?

```suggestion
stable release as well. Check the MSRV job of [the CI script](.github/workflows/main.yml) to see the oldest
```

Reply:
@ehuss that makes totally sense! Thank you for your help, I've applied the suggestion 👍